vllm/model_executor/models/mixtral.py

```diff
@@ -73,10 +75,31 @@ def __init__(self,
                  quant_config: Optional[QuantizationConfig] = None,
                  tp_size: Optional[int] = None,
                  dp_size: Optional[int] = None,
-                 prefix: str = ""):
+                 prefix: str = "",
+                 enable_eplb: bool = False):
         super().__init__()
         self.hidden_size = hidden_size
 
+        self.ep_group = get_ep_group().device_group
+        self.ep_rank = self.ep_group.rank()
+        self.ep_size = self.ep_group.size()
+
+        # Expert Parallelism Load balancing settings.
+        vllm_config = get_current_vllm_config()
+        parallel_config = vllm_config.parallel_config
+        self.enable_eplb = enable_eplb
+
+        self.n_routed_experts = num_experts
+        self.n_logical_experts = num_experts
+        self.n_redundant_experts = parallel_config.num_redundant_experts
```

Comment:
```suggestion
        self.n_redundant_experts = parallel_config.eplb_config.num_redundant_experts
```

We've switch to a new EPLB config interface in [config/parallel.py#L108-L111](https://github.com/vllm-project/vllm/blob/8ef6b8a38cc3b69e55b0bb60d5442c4efb1988de/vllm/config/parallel.py#L108-L111)